routes/api.php

```diff
@@ -17,3 +17,7 @@
 Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+Route::middleware('auth:sanctum')->get('/admin', function (Request $request) {
+    return $request->user();
+});
```

Comment:
block comment
i suggest these changes
```suggestion
Route::middleware('auth:sanctum')->get('/admins', function (Request $request) {
    // return the Admin user:
    // auto indentation in the comment box, nice!
    return $request->user();
});
```